website/docs/core-concepts/stacks/yaml-functions/include.mdx

```diff
@@ -38,6 +169,135 @@ The `!include` function can be called with either one or two parameters:
     <dd>(Optional) [YQ](https://mikefarah.gitbook.io/yq) expression to retrieve individual values from the file</dd>
 </dl>
 
+## Remote Includes
+
+Atmos uses [`go-getter`](https://github.com/hashicorp/go-getter)
+(used by [Terraform](https://www.terraform.io/) for downloading modules) to download remote files from different sources.
+
+### Downloading from OCI Registries
+
+The following example shows how to include a file from a public AWS ECR registry:
+
+  ```yaml
+    components:
+      terraform:
+        my-component:
+          vars: !include oci://public.ecr.aws/cloudposse/components/terraform/stable/aws/my-component.tfvars
```

Comment:
We shouldn't use this example any more because the repo was deleted.

Reply:
I think the most common would be from a repo, not from OCI. We should provide the most helpful/practical example first.